src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [peopleList, setPeopleList] = useState<Person[]>();
+  const [errorMessage, setErrorMessage] = useState(false);
+
+  const callRequest = async () => {
+    try {
+      const peopleFromServer = await getPeople();
+      const deepCopy = peopleFromServer.map(person => ({
+        ...person,
+        mother: peopleFromServer.find(
+          mother => mother.name === person.motherName,
+        ),
+        father: peopleFromServer.find(
+          father => father.name === person.fatherName,
+        ),
+      }));
+
+      setPeopleList(deepCopy);
+    } catch {
+      setErrorMessage(true);
+    }
+  };
+
+  useEffect(() => {
+    callRequest();
+  }, []);
```

Comment:
Why did you move callRequest to a separate function?

```suggestion
  useEffect(() => {
  const callRequest = async () => {
    try {
      const peopleFromServer = await getPeople();
      const deepCopy = peopleFromServer.map(person => ({
        ...person,
        mother: peopleFromServer.find(
          mother => mother.name === person.motherName,
        ),
        father: peopleFromServer.find(
          father => father.name === person.fatherName,
        ),
      }));

      setPeopleList(deepCopy);
    } catch {
      setErrorMessage(true);
    }
  };

    callRequest();
  }, []);
```

Why do so?
- Now linter can check your code for the correct useEffect dependencies
- No need to re-cereate the function on each component render
- If the function is relatively simple (like yours) the code remains readable